jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -860,6 +860,7 @@ pipeline {
                     archiveArtifacts artifacts: 'buildInfo.yml'
                     closeBuildSuccessGithubIssue(
                         message: buildMessage(search: 'Successfully built'),
+                        search: "Successfully built",
```

Comment:
Did not understand why 2 searches.

Reply:
Both are generics 2 libraries Sayali, `buildMessage` will give the entire log line starting with `Successfully built`, `search` will now get the component after `Successfully built + <component>`

It can be even be as follows 
```
closeBuildSuccessGithubIssue(
    message: buildMessage(search: 'Successfully Completed the component common-utils'),
    search: "component",
    inputManifestPath: "manifests/$INPUT_MANIFEST"
)
  ```
  
 So 1st you filter on a message (log line) and then filter for a component.

Reply:
Sayali, I will observe the runs with this library.